src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -138,7 +158,10 @@ private boolean isWhiteSpace(String letter) {
     // todo: elegance improvement: switch spelling to a stack. We do a lot of peeking
     // todo: elegance improvement: create an object that holds a reference to the spelling string and can do all this validation
     private boolean isCompleteToken(String spelling) {
-        if (validSolo.contains(spelling)) {
+        if(spelling.length()<1){
```

Comment:
This will be refactored to a stack, and I'm pretty sure it will never come up using the main branch implementation of scan.